Badge/style.css

```diff
@@ -0,0 +1,46 @@
+@import url(../reset.css);
+@import url(../Avatar/style.css);
+@import url(../index.css);
+
+.badge-container {
+  position: relative;
+  width: max-content;
+}
+
+.badge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  top: -4px;
+  right: -9px;
+  position: absolute;
+  font-size: 0.5rem;
+  color: white;
+  font-weight: 900;
+  height: 18px;
+  width: 18px;
+  border: none;
+  background-color: var(--danger-color);
+  border-radius: 50%;
+
+  height: 20px;
+  width: 20px;
```

Comment:
**Fix:** Height and width are mentioned twice here on lines 20-21 and 26-27. Remove the one which is not required.